scripts/globals/events/login_campaign.lua

```diff
@@ -0,0 +1,229 @@
+------------------------------------
+-- Login Campaign
+------------------------------------
+require("scripts/globals/npc_util")
+------------------------------------
+
+tpz = tpz or {}
+tpz.events = tpz.events or {}
+tpz.events.loginCampaign = tpz.events.loginCampaign or {}
+
+-- Change vars below to modify settings for current login campaign
```

Comment:
Do we have a history of data that could be applied?  I think this would be an excellent opportunity to try to figure out that pattern, and perhaps table it.

Reply:
@claywar  I wish there was a pattern, here's a link to all of the login campaigns:
https://www.bg-wiki.com/ffxi/Repeat_Login_Campaign/Past_Login_Campaign_Rewards#Repeat_Login_Campaign_No._1